TCAT/Controllers/RouteDetail+DrawerViewController.swift

```diff
@@ -51,22 +51,29 @@ class RouteDetailDrawerViewController: UIViewController {
     private var busDelayNetworkRefreshRate: Double = 10
     private var busDelayNetworkTimer: Timer?
     private let chevronFlipDurationTime = 0.25
+    /// Returns the currently expanded cell, if any
+    var expandedCell: LargeDetailTableViewCell? {
+        for index in 0..<tableView.numberOfRows(inSection: 0) {
+            let indexPath = IndexPath(row: index, section: 0)
+            if let cell = tableView.cellForRow(at: indexPath) as? LargeDetailTableViewCell {
+                if cell.isExpanded {
```

Comment:
can combine this with line 58:
`if let cell = tableView.cellForRow(at: indexPath) as? LargeDetailTableViewCell, cell.isExpanded {...}`